cmd/lncli/commands.go

```diff
@@ -1314,15 +1314,24 @@ var walletBalanceCommand = cli.Command{
 	Name:     "walletbalance",
 	Category: "Wallet",
 	Usage:    "Compute and display the wallet's current balance.",
-	Action:   actionDecorator(walletBalance),
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "account",
+			Usage: "The account for which the balance is shown.",
```

Comment:
nit: I think in the flag description we (usually, unfortunately not 100% consistently) we start lowercase and don't have full stop at the end of the sentences. Also, I'd add the `(optional) the account ...` prefix to make it more obvious it doesn't need to be specified.